packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'framework.dart';
+
+/// Embeds an Android view in the Widget hierarchy.
+///
+/// Embedding Android views is an expensive operation and should be avoided when possible.
```

Comment:
May need a bit more elaboration like what's the proposed alternative or why this exists (i.e. what are reasonable use cases)